libraries/esp8266/examples/LowPowerDemo/LowPowerDemo.ino

```diff
@@ -0,0 +1,365 @@
+      if (WiFi.localIP())  // don't check OTA if WiFi isn't connected
+        ArduinoOTA.handle();  //see if we need to reflash
+      delay(delayTime);
+    }
+  }
+  delay(50);  // debounce time for the switch, button pressed
+  while (!digitalRead(WAKE_UP_PIN))  // now wait for them to release the button
+    delay(10);
+  delay(50);  // debounce time for the switch, button released
+}
+
+uint32_t calculateCRC32(const uint8_t *data, size_t length) {
+  uint32_t crc = 0xffffffff;
+  while (length--) {
+    uint8_t c = *data++;
+    for (uint32_t i = 0x80; i > 0; i >>= 1) {
+      bool bit = crc & 0x80000000;
+      if (c & i) {
+        bit = !bit;
+      }
+      crc <<= 1;
+      if (bit) {
+        crc ^= 0x04c11db7;
+      }
+    }
+  }
+  return crc;
+}
+
+void updateRTC() {
+  rtcData.data[3] = resetLoop;  // save the loop count for the next reset
+  // Update CRC32 of data
+  rtcData.crc32 = calculateCRC32((uint8_t*) &rtcData.data[0], sizeof(rtcData.data));
+  if (resetLoop == 5)  // wipe the CRC in RTC memory when we're done with all tests
+    rtcData.crc32 = 0;
+  // Write struct to RTC memory
+  ESP.rtcUserMemoryWrite(64, (uint32_t*) &rtcData, sizeof(rtcData));
+}
+
+void init_WiFi() {
+  /* Explicitly set the ESP8266 as a WiFi-client (STAtion mode), otherwise by default it
+    would try to act as both a client and an access-point and could cause network issues
+    with other WiFi devices on your network. */
+  digitalWrite(LED, LOW);  // give a visual indication that we're alive but busy
+  WiFi.persistent(false);  // don't store the connection each time to save wear on the flash
+  WiFi.mode(WIFI_STA);
+  WiFi.config(staticIP, gateway, subnet);  // if using static IP, enter parameters at the top
+  WiFi.begin(AP_SSID, AP_PASS);
+  Serial.print(F("connecting to WiFi "));
+  Serial.println(AP_SSID);
+  DEBUG_PRINT(F("my MAC: "));
+  DEBUG_PRINTLN(WiFi.macAddress());
+  while (WiFi.status() != WL_CONNECTED)
+    delay(50);
+  DEBUG_PRINTLN(F("WiFi connected"));
+  while (!WiFi.localIP())
+    delay(50);
+  WiFi.setAutoReconnect(true);
+  DEBUG_PRINT(F("WiFi Gateway IP: "));
+  DEBUG_PRINTLN(WiFi.gatewayIP());
+  DEBUG_PRINT(F("my IP address: "));
+  DEBUG_PRINTLN(WiFi.localIP());
+}
+
+void init_OTA() {
+  // Port defaults to 8266
+  // ArduinoOTA.setPort(8266);
+
+  // Hostname defaults to esp8266-[ChipID]
+  // ArduinoOTA.setHostname("myesp8266"));
+
+  // No authentication by default
+  // ArduinoOTA.setPassword((const char *)"123"));
+
+  ArduinoOTA.onStart([]() {
+    Serial.println(F("Start"));
+  });
+  ArduinoOTA.onEnd([]() {
+    Serial.println(F("\nEnd"));
+  });
+  ArduinoOTA.onProgress([](unsigned int progress, unsigned int total) {
+    Serial.printf("Progress: %u%%\r", (progress / (total / 100)));
+  });
+  ArduinoOTA.onError([](ota_error_t error) {
+    Serial.printf("Error[%u]: ", error);
+    if (error == OTA_AUTH_ERROR) Serial.println(F("Auth Failed"));
```

Comment:
if-else bodies on their own lines, please, like so:
```cpp
if (this)
  doThat();
else if(this2)
  doThat2();
else
...
```

Reply:
Already fixed, just not committed yet.  I saw the error at Travis.